pkg/specgen/generate/kube/kube.go

```diff
@@ -182,6 +183,19 @@ func ToSpecGen(ctx context.Context, opts *CtrSpecGenOptions) (*specgen.SpecGener
 		if imageData.Config.WorkingDir != "" {
 			s.WorkDir = imageData.Config.WorkingDir
 		}
+		if s.User == "" {
+			s.User = imageData.Config.User
+		}
```

Comment:
Should we really be setting the user to the value found in the image?  That sounds like it might get twisted, or am I misreading this?